deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: storetheindex
+
+resources:
+  - ../../../../../base/assigner
+  - ingress.yaml
+
+patchesStrategicMerge:
+  - deployment.yaml
+
+secretGenerator:
+  - name: identity
+    behavior: replace
+    files:
+      - identity.key=indexer-0-identity.key.encrypted # 12D3KooWDBjcDRQ7CKJeF9Yy3UKbriHfyETDrXKzTDB6biH3ibBd
+
+configMapGenerator:
+  - name: config
+    behavior: replace
```

Comment:
This won't wok, because, `replace` expects a pre-existing config. The base `kustomization` did not define any. See earlier comment on `config` mounted volume.

Reply:
Removed. Do I need anything to specify the configuration file as a resource?